hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -416,7 +416,7 @@ public final class OzoneConfigKeys {
   public static final long HDDS_RATIS_SNAPSHOT_THRESHOLD_DEFAULT =
       ScmConfigKeys.HDDS_RATIS_SNAPSHOT_THRESHOLD_DEFAULT;
   public static final String HDDS_RATIS_LEADER_FIRST_ELECTION_MINIMUM_TIMEOUT_DURATION_KEY =
-      "hdds.ratis.leader.first.election.minimum.timeout.duration";
+      "hdds.ratis.leader.first-election.minimum.timeout.duration";
```

Comment:
@adoroszlai 
This reference is `HDDS_RATIS_LEADER_ELECTION_MINIMUM_TIMEOUT_DURATION_KEY`, which contains `minimum` and `duration`, but this is different style with `ScmConfigKeys.OZONE_SCM_HA_RATIS_LEADER_FIRST_ELECTION_TIMEOUT`

Should we remove the `minimum` and `duration` ?

Reply:
We also have:

https://github.com/apache/ozone/blob/ade69e3f821ea6c6a5ff199c8aac7987e44d48cf/hadoop-hdds/common/src/main/resources/ozone-default.xml#L2201